### PR TITLE
OP-138 load patient profile picture from file system

### PR DESCRIPTION
**Pull request by AndreiDodu**
See OP-138.

Paired with https://github.com/informatici/openhospital-core/pull/692.

### CODE REVIEW

File: src/main/java/org/isf/utils/image/ImageUtil.java [hunk 1]
```diff
@@ -74,5 +90,32 @@ public static byte[] imageToByte(final BufferedImage bufferedImage) {
...
+	}
+	
+	
+	public static BufferedImage fixImageFileSize(BufferedImage bufferedImage, int maximumFileSize, String fileType) throws IOException {
+		long arrSize = getArraySize(bufferedImage, fileType);
+		while (arrSize > maximumFileSize) {
+			int newTargetSize = (bufferedImage.getTileWidth()- ((bufferedImage.getTileWidth() / 100) * 10));
+			bufferedImage = Scalr.resize(bufferedImage, newTargetSize);
+			arrSize = getArraySize(bufferedImage, fileType);
+			}
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/utils/image/ImageUtil.java:105]: strange formatting; shouldn't the brace be lined up under the `while`?

File: bundle/language_it.properties [hunk 2]
```diff
@@ -1175,7 +1175,7 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = I genitori sono insieme
 angal.patient.patientbrowser.title                                                                     = Elenco Pazienti
 angal.patient.patientphoto                                                                             = Foto
-angal.patient.patientselection.title                                                                   = Selezione Paziente
+angal.patient.patientselection.title                                                                   = Non � stata trovata la foto del paziente.
```
Comment (mwithi) [inline bundle/language_it.properties:1178]: There's no need to add new keys in the language other than "en". Please revert this change.

**Review by mwithi — APPROVED** [review]
Approved all changes, merged into the temporary branch `OP-138_Patient_Photo_to_FS `
Thanks!
